reverse-proxy.md

```diff
@@ -254,9 +254,8 @@ Of course you need to modify `<your-nc-domain>` to the domain on which you want
 
 <summary>click here to expand</summary>
 
-**Disclaimer:** It might be possible that the config below is not working 100% correctly, yet. Improvements to it are very welcome!
-
-Add this to you nginx config:
+Add this to you nginx config
+**Note:** please check your nginx version by running: `nginx -v` and adjust it the lines marked with version notes, so that they fit your nginx version
```

Comment:
probably needs a <br> here to separate lines?

Reply:
What do you mean?

Reply:
```suggestion
Add this to you nginx config <br>
**Note:** please check your nginx version by running: `nginx -v` and adjust it the lines marked with version notes, so that they fit your nginx version
```
do you mean this?

Reply:
yes